### PR TITLE
fix: fix docs deploy

### PR DESCRIPTION
**Pull request by dhmlau**
Signed-off-by: Diana Lau <dhmlau@ca.ibm.com>

### CODE REVIEW

File: .github/workflows/site-build.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - gh-pages
+  pull_request:
```
Comment (dhmlau) [inline .github/workflows/site-build.yml]: this is for testing.

**Comment by dhmlau** [comment]
@raymondfeng @achrinza, I tried it with my own fork that the only change that's needed to fix the LB4 docs issue is to use `npm install` instead of `npm ci`, so I'm removing the deploy portion of it.

File: .github/workflows/pr-build.yml [hunk 2]
```diff
@@ -25,9 +25,9 @@ jobs:
...
-          bundle install
-          npm install
-          npm ls @loopback/docs 2>/dev/null || true
-          npm run build
+#      - name: Install and Build 🔧
+#        run: |
+#          bundle install
+#          npm install
+#          npm ls @loopback/docs 2>/dev/null || true
+#          npm run build
```
Comment (raymondfeng) [inline .github/workflows/pr-build.yml]: I wonder why you commented the build step for the PR. The commands will NOT run with gh-pages branch and they should not mess up.

Comment (dhmlau) [reply .github/workflows/pr-build.yml]: you're right. At first, i thought we may not need this but I guess it's needed to make sure the web site can build. Will add it later.
I tested it again with the deploy action. https://github.com/dhmlau/loopback.io/blob/gh-pages/.github/workflows/site-build.yml#L35-L40. Will do more testing again on my repo first and update this PR.

**Comment by dhmlau** [comment]
Tested the changes with my fork: https://github.com/dhmlau/loopback.io/tree/gh-pages/.github/workflows.

**Review by raymondfeng — APPROVED** [review]

**Pull request merged**